qiita_db/analysis.py

```diff
@@ -79,6 +79,26 @@ def _status_setter_checks(self, conn_handler):
             raise QiitaDBStatusError("Can't set status away from public!")
 
     @classmethod
+    def get_user_default(cls, user):
```

Comment:
That call is executed when you are doing `cls(aid)` (It is calling the __init__)